packages/core/src/config/recommended.ts

```diff
@@ -90,6 +91,7 @@ const recommended: PluginStyleguideConfig<'built-in'> = {
   async2Rules: {
     spec: 'error',
     'info-contact': 'off',
+    'info-license-strict': 'warn',
```

Comment:
I'm not sure I would make this change to the recommended ruleset if the old rule wasn't already here (also for AsyncAPI 3)

Reply:
It just happens to cover the same structure requirements as in OAS3/3.1, so I decided to include it to align more closely with our OAS ruleset. I don't see how a recommended rule for the OAS info section could be harmful for AsyncAPI, especially considering that we only recently added it and most likely just forgot to include the rule in AsyncAPI's ruleset. It's still a warning anyway.